whisper/whisperv6/message.go

```diff
@@ -124,6 +124,8 @@ func (msg *sentMessage) appendPadding(params *MessageParams) error {
 	rawSize := len(params.Payload) + 1
 	if params.Src != nil {
 		rawSize += signatureLength
+	} else {
+		rawSize += AESNonceLength
```

Comment:
this is true only for symmetric encryption

Reply:
If `Src` is `nil`, then it's using symmetric encryption

Reply:
Src is used to identify the source of the message, i.e. for digital signature. Dst is used for asymmetrc encryption. And KeySym is used for symmetric encryption. So, you can change line 127 like this:
} else if params.KeySym != nil {

Reply:
Well then it's very confusing. Then both sym and asym should be able to sign their messages. So it should be `if` and not `else if`